microprofile/metrics/src/main/java/io/helidon/microprofile/metrics/InterceptorSimplyTimed.java

```diff
@@ -29,11 +38,15 @@
  *     method, and the separate {@link InterceptorSyntheticRestRequest} interceptor deals with those.
  * </p>
  */
-@SimplyTimed
+@InterceptorSimplyTimed.Binding
 @Interceptor
 @Priority(Interceptor.Priority.PLATFORM_BEFORE + 10)
 final class InterceptorSimplyTimed extends InterceptorTimedBase<SimpleTimer> {
 
+    static Binding.Literal binding() {
```

Comment:
Static methods should be after constructors. This is valid for all interceptors.

Reply:
Changed